learn/03_agile_lean.md

```diff
@@ -31,3 +31,4 @@ Week Goal: To deepen your understanding of agile and lean principles and how the
 * Cutting code on the greenfield application
 * Retrospective (review action points, create improvements for next week)
 * Showcase to the entire company
```

Comment:
Is this a brutal thing to ask for the first week? I think it's really important, but wonder if we could defer it until the second week and helps to have consistency around the showcase. I'd also include some practice showcases within the general planning, so they can build some confidence around this and so it's got some structure